frontend/src/components/Signup/Signup.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import { Animated } from 'react-animated-css';
+import { Input, Icon, Button } from 'antd';
+import './Signup.css';
+
+const circleLogo = require('../../assets/imgs/circle-logo.svg');
+
+class Signup extends Component {
+  state = {
+    email: '',
+    profile: '',
+    password1: '',
+    password2: '',
+    firstName: '',
+    lastName: '',
+    address: '',
+    phoneNumber: '',
+    dateOfBirth: ''
+  }
+
+  handleChange = (e) => {
+    this.setState({[e.target.name]: e.target.value});
+  }
+
+  render(){
+    return(
+      <div className="signup__background">
+        <Animated animationIn="fadeInLeft" isVisible={true}>
+          <div className="container-fluid signup__removePadding">
+            <div className="row signup__removeMargin">
+              <div className="col-6 signup__removePadding signup__suLeft">
```

Comment:
Can we use a `Form` component from `antd`? That way, we have access to an `onSubmit` function instead of updating state each time.

Reply:
@ericong18 Could you take a look at the PR again? The changes have been made.

Reply:
In `handleSubmit`, you have access to an event variable, so you can access the value in the input field with `e.target.[name].value`. Although this requires you to have an `name` prop in `Input`.